message/outbound_msg_builder.go

```diff
@@ -284,13 +313,13 @@ func (b *outMsgBuilder) PeerList(peers []ips.ClaimedIPPort, bypassThrottling boo
 			IpPort:          uint32(p.IPPort.Port),
 			Timestamp:       p.Timestamp,
 			Signature:       p.Signature,
-			TxId:            p.TxID[:],
+			TxId:            ids.Empty[:],
```

Comment:
This must still be populated as other nodes expect to be able to parse it. Note: Sending 32 bytes of zeros here doesn't cause nodes to drop the message.